components/shared/icon/icon.tsx

```diff
@@ -3,21 +3,24 @@ import classNames from "classnames";
 
 import style from './icon.module.scss';
 
-type VariantType = "information" 
+export type IconVariantType = "information" 
+| "attachment"
 | "bubble-speech"
-| "mail"
-| "user"
-| "search"
 | "chat"
-| "remove"
+| "edit"
+| "information"
+| "lock"
+| "mail"
 | "plane"
 | "power"
-| "attachment"
-| "lock"
-| "edit"
+| "remove"
+| "search"
+| "user"
+
+
 
 type Props = {
-    variant: VariantType,
+    variant: IconVariantType,
     className?: string,
 }
```

Comment:
In types we set semicolon in the line end  ⬇️ 

```suggestion
    variant: IconVariantType;
    className?: string;
}
```